.aios-core/infrastructure/scripts/test-discovery.js

```
@@ -717,21 +717,12 @@ class CoverageAnalyzer {
           if (match) {
             try {
-              // Try to parse as JSON first (safest)
-              // Convert JS object literal to valid JSON by adding quotes to keys
-              const jsonLike = match[1]
-                .replace(/(['"])?([a-zA-Z0-9_]+)(['"])?\s*:/g, '"$2":')
-                .replace(/'/g, '"')
-                .replace(/,\s*}/g, '}')
-                .replace(/,\s*]/g, ']');
-              return JSON.parse(jsonLike);
+              // eslint-disable-next-line no-eval
+              return eval('(' + match[1] + ')');
```

Biome (2.3.13)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)